04-path-security-and-networking/405-ingress-controllers/readme.adoc

```diff
@@ -333,11 +329,7 @@ and add this to your node policy:
           "Action": [
             "acm:ListCertificates",
             "acm:DescribeCertificate",
-            "autoscaling:DescribeAutoScalingGroups",
-            "autoscaling:AttachLoadBalancers",
-            "autoscaling:DetachLoadBalancers",
-            "autoscaling:DetachLoadBalancerTargetGroups",
-            "autoscaling:AttachLoadBalancerTargetGroups",
+            "autoscaling:*",
```

Comment:
Same feedback, if there is a specific permission that is missing, I'd be curious to know what it is so that we can educate better.